x-pack/plugins/apm/public/components/app/RumDashboard/UXMetrics/index.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { I18LABELS } from '../translations';
+import { CoreVitals } from '../CoreVitals';
+import { KeyUXMetrics } from './KeyUXMetrics';
+import { useUrlParams } from '../../../../hooks/useUrlParams';
+import { useFetcher } from '../../../../hooks/useFetcher';
+
+export interface UXMetrics {
+  cls: string;
+  fid: string;
+  lcp: string;
+  tbt: string;
+  fcp: number;
+  lcpRanks: number[];
+  fidRanks: number[];
+  clsRanks: number[];
+}
+
+export function UXMetrics() {
+  const { urlParams, uiFilters } = useUrlParams();
+
+  const { start, end } = urlParams;
+
+  const { data, status } = useFetcher(
+    (callApmApi) => {
+      const { serviceName } = uiFilters;
+      if (start && end && serviceName) {
+        return callApmApi({
+          pathname: '/api/apm/rum-client/web-core-vitals',
+          params: {
+            query: { start, end, uiFilters: JSON.stringify(uiFilters) },
+          },
+        });
+      }
+      return Promise.resolve(null);
+    },
+    [start, end, uiFilters]
+  );
+
+  return (
+    <EuiPanel>
+      <EuiFlexGroup justifyContent="spaceBetween">
+        <EuiFlexItem grow={1} data-cy={`client-metrics`}>
+          <EuiTitle size="s">
+            <h3>{I18LABELS.userExperienceMetrics}</h3>
+          </EuiTitle>
```

Comment:
```suggestion
          <EuiTitle size="xs">
            <h3>{I18LABELS.userExperienceMetrics}</h3>
          </EuiTitle>
```

Probably a small visual fix, but it looks like there's a difference in the panel titles, but in the code they're all `h3`'s. If we intend to have section header vs. panel titles, we should probably think about the semantic structure here.

![93121454-25b79580-f6c5-11ea-84e6-5716c3712efb](https://user-images.githubusercontent.com/4104278/93311018-c0b78900-f805-11ea-8ac5-7d05c4c1c1cc.png)

A small change would be just to lower the title size of that "User experience metrics" title.